zerver/views/webhooks/trello/board_actions.py

```diff
@@ -0,0 +1,107 @@
+from typing import Dict, Any, Tuple
+from .exceptions import UnknownUpdateBoardAction
+from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
+
+SUPPORTED_BOARD_ACTIONS = [
```

Comment:
In `zerver.views.webhooks.trello.api_trello_webhook`, we'll get `action_type` be decoding JSON and doing a dictionary lookup. So it will be of type `text_type`. So for consistency we should make all `SUPPORTED_BOARD_ACTIONS` `text_type` strings.